contrib/scripts/check-time.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
```

Comment:
Do any of those options provide a way to script updating the dependency?

Reply:
+1 on proceeding since the script is functional but it might be a good idea to capture the Go/gomodguard refactor in an issue.

Reply:
> Do any of those options provide a way to script updating the dependency?

No it would only cover the linting (check) part

> At this point I think this is functional enough that I'd like to move on.

Totally fine for me :+1: Just thought it's worth to drop a comment

Reply:
Yep good feedback and worth considering. Thanks for raising this discussion item.